mobile-app/app/screens/AppNavigator/screens/Settings/screens/LanguageSelectionScreen.tsx

```diff
@@ -2,22 +2,25 @@ import { ThemedScrollView } from '@components/themed'
 import { ThemedSectionTitle } from '@components/themed/ThemedSectionTitle'
 import { getAppLanguages, translate } from '@translations'
 import { RowLanguageItem } from '../components/RowLanguageItem'
+import { tailwind } from '@tailwind'
 
 export function LanguageSelectionScreen (): JSX.Element {
   const languages = getAppLanguages()
 
   return (
     <ThemedScrollView testID='language_selection_screen'>
       <ThemedSectionTitle
+        style={tailwind('px-8 pt-6 pb-2 text-xs text-gray-500 font-medium')}
```

Comment:
screen background looks different from Figma. Need to double check

Reply:
check if its correct already :v